CSS/button.css

```diff
@@ -0,0 +1,58 @@
+.btn-primary{
+    background-color: var(--primary-dark-color) !important;
+    font-size: 1.5rem !important;
+}
+
+.btn-secondary{
+    background-color: var(--secondary-color) !important;
+    font-size: 1.5rem !important;
+}
+
+.btn-outline-primary{
+    border: 2px solid var(--primary-dark-color) !important;
+    color: var(--primary-dark-color) !important;
+    background-color: var(--black) !important;
+    font-size: 1.5rem !important;
+}
+
+.btn-outline-primary:hover{
+    background-color: var(--primary-dark-color) !important;
+    color: var(--white) !important;
+}
+
+.btn-outline-secondary{
+    border: 2px solid var(--secondary-color) !important;
+    color: var(--secondary-color) !important;
+    background-color: var(--white) !important;
+    font-size: 1.5rem !important;
+
+}
+
+.btn-outline-secondary:hover{
+    background-color: var(--secondary-color) !important;
+    color: var(--white) !important;
+}
```

Comment:
you are using to many "important" maybe you can increase the specificity that would work

Reply:
I used component from css component library so that to overwrite some style I used it